rfc/php_technical_committee.wiki

```diff
@@ -0,0 +1,102 @@
+====== PHP RFC: PHP Technical Committee ======
+  * Version: 0.1
+  * Date: 2023-03-03
+  * Author: Jakub Zelenka, bukka@php.net
+  * Status: Draft
+
+===== Introduction =====
+
+PHP uses voting system for deciding about changes to PHP. This was created mainly for user visible changes in language.
+It works well for that purpose but it is not that effective for deciding about purely technical changes that impact PHP
+internals and extension API's. It is not exactly clear how it should resolve conflicts (on technical bases) between
+contributors that sometimes happen.
+
+
+===== Proposal =====
+
+This RFC proposes introduction of the PHP Technical Committee (TC) that would decide about all technical aspects of
+php-src. Specifically its main purpose is to decide about disagreements between contributors but it is not limited to
+that only.
+
+==== Definitions ====
+
+The text contains various terms that need further clarification to prevent future wording dispute. Here are their
+definitions:
+
+  * contributor - Anyone who has already contributed to php-src.
+  * core developer - Anyone with write commit right to php-src.
+  * dev branch - php-src branch consisting of PHP- prefix and major and minor version number (e.g. PHP-8.2)
+  * issue - php-src GitHub issue.
+  * php-src - Source code in https://github.com/php/php-src/.
+  * PR - Pull request.
+  * processing changes - Any changes impacting the way how PHP project is managed and its bylaws.
+  * pull request - php-src GitHub pull request.
+  * RM - PHP release managers.
+  * technical aspect - Technical side of any issue or pull request in php-src or commit to php-src master and dev branches.
+  * technical descision - Decision on a technical aspect.
+  * technical side - All php-src changes excluding user facing changes. It covers alse the implementation of user facing changes.
+  * TC - PHP Technical Committee.
+  * user facing changes - Changes in PHP languege, user visible funtions, classes, interfaces, constants and other user visible constructs.
```

Comment:
The definitions of technical aspect and user-facing changes could use fleshing out more, preferably with examples.

Reply:
Added few examples. Is it better?